src/cli.js

```diff
@@ -130,6 +134,14 @@ export async function initProject(config) {
     }
   }
 
+  // push files to github
+  if (config.tooling.sourceControl === constants.github.name) {
```

Comment:
This should be dependent upon `createGithubRepo()` succeeding. Same with enabling Travis. See comment below for more info.

Reply:
Addressed